appveyor.yml

```diff
@@ -1,25 +1,29 @@
-version: 13.1.0-ci-{build}
+version: 13.2.0-ci-{build}
 pull_requests:
   do_not_increment_build_number: true
 branches:
   except:
     - gh-pages
+    - docs
 image: Visual Studio 2017
 configuration: Debug
 before_build:
   - nuget restore Telegram.Bot.sln
 build:
+  project: Telegram.Bot.sln
   publish_nuget: true
   publish_nuget_symbols: true
   verbosity: normal
 after_build:
+  - ps: "dotnet test test/Telegram.Bot.Tests.Unit/Telegram.Bot.Tests.Unit.csproj --list-tests"
+  - ps: "dotnet test test/Telegram.Bot.Tests.Unit/Telegram.Bot.Tests.Unit.csproj --no-build"
   - ps: "Get-ChildItem *.nupkg -Recurse | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }"
 test: off
 deploy:
   - provider: NuGet
-    server: https://www.myget.org/F/telegram-bot/api/v2/package
+    server: https://www.myget.org/F/telegram-bot-core/api/v2/package
     api_key:
-      secure: Dl2/cPfGdESLPBMd7Y+CIQomwOgNQ2+rUTylGGzMb0FaJaL4WgPPf5lzkY5KD4V6
-    symbol_server: https://www.myget.org/F/telegram-bot/api/v2/package
+      secure: gSQ82eM3mJYkPfEF2qPLvzEw45dS6C/VAwLHTMn3mRpVMhVybLzBUCWDnP5lAmp6
+    symbol_server: https://www.myget.org/F/telegram-bot-core/api/v2/package
```

Comment:
@MrRoundRobin 
> is this ok to be released before work on API v3.3?

I published 2 versions to `.Core` package so far and think it's better to publish the same versions(see tags and milestones) for original package as well. It wouldn't make users confused about what fix is in what version.